monkeytype/db/base.py

```diff
@@ -71,7 +71,8 @@ def __init__(self, store: CallTraceStore) -> None:
         self.traces: List[CallTrace] = []
 
     def log(self, trace: CallTrace) -> None:
-        self.traces.append(trace)
+        if not trace.func.__qualname__.startwith('__main__'):
```

Comment:
As typecheck on TravisCI is pointing out, this is a typo, should be `__startswith__`.

Also, I think we need to be checking `__module__` not `__qualname__`; `__qualname__` is a bit confusingly named, it's only "qualified" within the module (e.g. includes class name), it doesn't include the module name.

I think these two issues point out the value of including a regression test for this functionality.